packages/terra-form-input/src/Input.module.scss

```diff
@@ -26,6 +26,15 @@
       color: var(--terra-form-input-placeholder-color, #70787c);
       font-style: var(--terra-form-input-placeholder-font-style);
     }
+    
+    /* TODO: Remove vendor prefix once we update to autoprefixer v9 */
+    /* stylelint-disable declaration-colon-space-after */
```

Comment:
This could be removed if you do -

`color: var(--terra-form-input-placeholder-color, #70787c);`

Reply:
Thanks..! i added this by thinking warning was for pseudo element selector ':-ms-input-placeholder'. Formatted as shown above : 893f5a1c5d4ff2fc96f0fc0684dec129b8cb55cc